src/components/common/Clickable.d.ts

```diff
@@ -1,9 +1,10 @@
+import React from 'react';
 import type { InteractiveProps } from 'src/components/Interactive';
 
 export type ClickableProps<T extends keyof JSX.IntrinsicElements = 'div'> = {
   as?: T;
   href?: string;
-  tooltip?: string;
+  tooltip?: string | React.ReactElement<'div'>;
```

Comment:
ReactNode would be a better type for this.

```suggestion
  tooltip?: ReactNode;
```